pkg/apis/options/providers.go

```diff
@@ -128,6 +128,9 @@ const (
 
 	// OIDCProvider is the provider type for OIDC
 	OIDCProvider ProviderType = "oidc"
+
+	// CidaasProvider is the provider type for Cidaas IDP
+	CidaasProvider ProviderType = "cidaas"
```

Comment:
This struct is alphabetically sorted. Please move the provider in-between bitbucket and DO